changelog/issue-2393.md

```diff
@@ -2,4 +2,6 @@ audience: developers
 level: minor
```

Comment:
I missed this on the earlier PR, but let's change this to "patch".  The guidelines for these levels are [here](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/changelog.md), In this case, "A patch change is one that fixes a bug or makes a minor change which users can do their work without." -- users can do their work without this feature, just a bit more slowly.